docs/source/reference/functions.rst

```diff
@@ -315,6 +315,10 @@ exp
 ~~~
 .. autofunction:: exp
 
+floor
+~~~
```

Comment:
This line is too short. It needs to have the same length with "floor", i mean it requires five characters.